src/System.Text.Json/src/System/Text/Json/JsonEncodedText.cs

```diff
@@ -35,38 +36,42 @@ private JsonEncodedText(byte[] utf8Value)
         /// Encodes the string text value as a JSON string.
         /// </summary>
         /// <param name="value">The value to be transformed as JSON encoded text.</param>
+        /// <param name="encoder">The encoder used to escape the input string.</param>
         /// <exception cref="ArgumentNullException">
         /// Thrown if value is null.
         /// </exception>
         /// <exception cref="ArgumentException">
         /// Thrown when the specified value is too large or if it contains invalid UTF-16 characters.
         /// </exception>
-        public static JsonEncodedText Encode(string value)
+        /// <remarks>By default, if no encoder is passed in, <see cref="JavaScriptEncoder.Default"/> is used</remarks>
```

Comment:
Based on https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.dictionary-2.-ctor?view=netframework-4.8#System_Collections_Generic_Dictionary_2__ctor_System_Collections_Generic_IEqualityComparer__0__ I think that the remark should be merged into the param doc.

```C#
/// <param name="encoder">The encoder to use when escaping the string, or <see langword="null" /> to use the default encoder.</param>
```

Reply:
Thanks. Updating the "params" and removing the "remarks" section (for the 3 instances of this).